### DISCUSSION TITLE
My website title shows "Untitled" on google

### DISCUSSION THREAD

**Discussion by hrhoo** [alshedivat/al-folio#2626] [Q&A] [answered]
My webstie shows up on google. However, it shows "Untitled". I am not sure where to change...

In "_config.yml, I have left title as "blank" and filled my name

**Comment by hrhoo** [comment]
When I check my first page source code,

`<title> my name </title> <meta name="author" content="my name">`

it is like this. I think my website title should not be "Untitled", but it is on google search.

**Reply by george-gca** [reply, marked as answer]
Google Search works on mysterious ways. It is possible that your site was indexed in a previous version where this was wrong, then after it was updated in your end Google hasn't updated it on its end.

Check https://github.com/alshedivat/al-folio/discussions/2585#discussioncomment-10097726 to see if it helps.

**Comment by alexander-reeves** [comment]
Any fix on this? I have the same issue and it looks bad.

**Reply by george-gca** [reply]
Check the above comment for more info on this, but there is only so much that can be done in our side.

**Comment by tropikalkimani** [comment]
> When I check my first page source code,
> 
> `<title> my name </title> <meta name="author" content="my name">`
> 
> it is like this. I think my website title should not be "Untitled", but it is on google search.
> 



True. The label at the browsers tab should read as you have written between the title tags. 

How about you try this . 

<title><p>My Name</p></title> 

Or let the name be a one-word text i.e. myName or my-name

**Reply by tropikalkimani** [reply]
Enclose the name of the title in p tags
